src/components/common/ColonyActionsTable/partials/ActionsTableFilters/partials/filters/DateFilters/DateFilters.tsx

```diff
@@ -52,6 +53,10 @@ const DateFilters: FC = () => {
             '!fixed !inset-auto !left-1/2 !top-1/2 w-full !-translate-x-1/2 !-translate-y-1/2':
               isMobile,
           })}
+          // I can't just use Object.values().some(Boolean) since I really need to specifically check if it's true, and not if it's truthy
+          disabled={Object.values(dateFilters).find(
+            (dateFilter) => dateFilter === true,
+          )}
```

Comment:
please review **disabled** flow again (I left a detailed comment above). thanks!

Reply:
Hey @Nortsova all your points are super valid! 🙌  I spoke to @arrenv and basically we'll have to create a separate issue to address all those things you've mentioned as they are currently on production as well

As a follow-up, I've also decided to remove the disabling of the fields because we might end up using radio buttons for the date filters, as per Arren's suggestions.

Thank you for reviewing this!

I think things to note now that the disable logics are removed: When a custom date range is entered, it will be prioritised. This is production behaviour.

I'll be adding this in the description as well 😄